src/docs/redis/sections/20-operationalize/20-root.mdx

```diff
@@ -0,0 +1,5 @@
+Here are some notes on root distribution for Redis.
+
+- "Adding SAN validation also makes sense (again, optionally) but will probably be more difficult to do, as Redis tends to use IPs rather than hostnames in all sorts of places. [https://github.com/redis/redis/issues/7946#issuecomment-714660881](https://github.com/redis/redis/issues/7946#issuecomment-714660881)
+    - "The best option is to disable matching of hostname to certificate name. As long as your CA is private the security implications should be minor" [https://github.com/redis/redis/issues/7928#issuecomment-714462007](https://github.com/redis/redis/issues/7928#issuecomment-714462007)
+- The `tls-ca-cert-dir` directive is complicated: [https://github.com/redis/redis/issues/7392](https://github.com/redis/redis/issues/7392)
```

Comment:
I notice that this is defined as an override for redis. This means that if we have some prose in the default `20-root.mdx` about using your MDM or ansible or whatever, then they won't show for Redis. Intentional, or should we define a second optional section for specifics on a technology?